modules/installation-about-custom-gcp-vpc.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-vpc.adoc
+
+[id="installation-about-custom-gcp-vpc_{context}"]
+= About using a custom VPC
+
+In {product-title} {product-version}, you can deploy a cluster into an existing Virtual Private Cloud (VPC) in Google Cloud Platform (GCP). If you do, you must also use existing subnets within the VPC and routing rules.
+
+By deploying {product-title} into an existing GCP VPC, you might be able to avoid service limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. This is a good option to use if you cannot obtain the infrastructure creation permissions that are required to create the VPC.
+
+[id="installation-about-custom-gcp-vpcs-requirements_{context}"]
+== Requirements for using your VPC
+
+When you deploy a cluster by using an existing VPC, you must perform additional network configuration before you install the cluster. In installer-provisioned infrastructure clusters, the installer usually creates the following components, but it does not create them when you install into an existing VPC:
+
+* VPC
+* Subnets
+* Cloud Router
+* Cloud NAT
+* NAT IP addresses
+
+If you use a custom VPC, you must correctly configure it and its subnets for the installation program and the cluster to use. The installation program cannot subdivide network ranges for the cluster to use, set route tables for the subnets, or set VPC options like DHCP, so you must do so before you install the cluster.
+
+Your VPC and subnets must meet the following characteristics:
+
+* The VPC’s CIDR block must contain the `Networking.MachineCIDR` range, which is the IP address pool for cluster machines.
+* The VPC must be in the same GCP project that you deploy the {product-title} cluster to.
+* You must configure Cloud NAT on the subnets to allow egress to the internet from the control plane and compute machines because these machines do not have any public address. Because multiple Cloud NATs cannot be configured on the shared subnets, the installation program cannot configure it.
+
+To ensure that the subnets that you provide are suitable, the installation program confirms the following data:
+
+* All the subnets that you specify exist.
```

Comment:
@kalexand-rh It's a little bit confused to me. I suggest to fix like this:
All the subnets that you specify exist and belong to the VPC you specify
The subnet CIDRs belong to the machine CIDR